src/scipp/core/like.py

```diff
@@ -23,15 +23,26 @@ def zeros_like(
     var: _Union[_cpp.Variable,
                 _cpp.DataArray]) -> _Union[_cpp.Variable, _cpp.DataArray]:
     """
-    Constructs a new object with the same dims, shape, unit and dtype as the input
-    (:class:`Variable` or :class:`DataArray`), but with all values initialized to 0.
+    Return a Variable or DataArray with the same dims, shape, unit, and dtype as the
+    input and all values initialized to 0.
+
     If the input has variances, all variances in the output are set to 0.
-    If the input is a :class:`DataArray`, coordinates and attributes are shallow-copied
+    If the input is a data array, coordinates and attributes are shallow-copied
     and masks are deep copied.
 
-    :param var: Input variable or data array.
+    Parameters
+    ----------
+    var:
+        Input object defining dims, shape, unit, and dtype of the output
+
+    Returns
```

Comment:
I most often find this `Returns` field a little pointless, because it almost always duplicates either info in the main description, or the return type (which is auto-generated).

That said, this is not specific to Napoleon, it also applies to the current way we generate the docs.